Cargo.toml

```diff
@@ -14,6 +14,8 @@ keywords = ["no-std", "NaCl", "Ed25519", "cryptography", "signatures"]
 cosey = { version = "0.3.0", optional = true }
 subtle = { version = "2.4.0", default-features = false }
 zeroize = { version = "1.2.0", default-features = false }
+ed25519 = { version = "1.3.0", default-features = false }
+signature = { version = "1.4.0", default-features = false }
```

Comment:
ed25519 re-exports signature, please use that so there are less dependency versions to track.